CharlieBackend.AdminPanel/Controllers/CalendarController.cs

```diff
@@ -0,0 +1,27 @@
+﻿using CharlieBackend.AdminPanel.Services.Interfaces;
+using CharlieBackend.Core.DTO.Schedule;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System.Threading.Tasks;
+
+namespace CharlieBackend.AdminPanel.Controllers
+{
+    [Authorize(Roles = "Admin")]
+    public class CalendarController : Controller
+    {
+        private readonly ICalendarService _calendarService;
+
+        public CalendarController(ICalendarService calendarService)
+        {
+            _calendarService = calendarService;
+        }
+
+        public async Task<IActionResult> Index(
+            ScheduledEventFilterRequestDTO scheduledEventFilter)
+        {
+            var data = await _calendarService.GetCalendarDataAsync(scheduledEventFilter);
+
+            return Ok(data);
+        }
```

Comment:
Temporary implementation that is to be changed in https://github.com/ita-social-projects/WhatBackend/issues/445.
Tested:
![tested](https://user-images.githubusercontent.com/38729881/110365677-f8cdb500-804d-11eb-8471-f9ae6c754465.png)